contracts/token/ERC1155/ERC1155.sol

```diff
@@ -3,13 +3,13 @@
 
 pragma solidity ^0.8.19;
 
-import "./IERC1155.sol";
-import "./IERC1155Receiver.sol";
-import "./extensions/IERC1155MetadataURI.sol";
-import "../../utils/Context.sol";
-import "../../utils/introspection/ERC165.sol";
-import "../../utils/Arrays.sol";
-import "../../interfaces/draft-IERC6093.sol";
+import {IERC1155} from "./IERC1155.sol";
+import {IERC1155Receiver} from "./IERC1155Receiver.sol";
+import {IERC1155MetadataURI} from "./extensions/IERC1155MetadataURI.sol";
+import {Context} from "../../utils/Context.sol";
+import {IERC165, ERC165} from "../../utils/introspection/ERC165.sol";
```

Comment:
In these case, do we want to duplicate the import to fetch from the original file ?

```suggestion
import {IERC165} from "../../utils/introspection/IERC165.sol";
import {ERC165} from "../../utils/introspection/ERC165.sol";
```

Reply:
I don't have a strong opinion, initially I thought there would be a lot more of these cases and that it would be better to have fewer import statements. But I think it's just a few places and we could do either.